plugins/ru.runa.gpd/src/ru/runa/gpd/ui/wizard/VariableDefaultValuePage.java

```diff
@@ -93,6 +170,15 @@ protected void verifyContentIsValid() {
                     if (Group.class.getName().equals(className) || Actor.class.getName().equals(className)
                             || Executor.class.getName().equals(className)) {
                         // TODO validate using connection?
+                    } else if (FileVariable.class.getName().equals(className)) {
+                    	if (isUseDefaultValue()) {
+	                    	if (isEmbeddedFileWithProtocol()) {
+	                    		file = EmbeddedFileUtils.getProcessFile(defaultValue);
```

Comment:
Unable to convert string to variable type: IFile 'processfile://7036.xlsx' is not exists!